samples/core-projects/libraries/README.md

```diff
@@ -0,0 +1,83 @@
+# .NET Core Library Samples
+
+**Note:** These are compatibile with tooling that is still in active development.  Details are subject to change over time.
+
+These samples are buildable projects whose source are used for code snippets in [the guide for writing cross-platform libraries](../../../docs/libraries/libraries-with-cli.md).  They can be built and run using the .NET Core RC1 toolchain.
+
+They are small and intended to demonstrate how to target and build NuGet packages for different targets.
+
+To build/use any of these (using `new-library` as an example):
+
+1. Open your favorite Command Line Interface.
```

Comment:
I'm good with this wording wise, but I'm wondering if some of our readers might not get what you mean. You might try:

> Open your favorite Command Line Interface (e.g. cmd.exe, Terminal).